Backend/BackendAPI/Repository/Interfaces/IGenericRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BackendAPI.Repository.Interfaces
+{
+    public interface IGenericRepository<T>//TODO: do rozważenia where T is CoreComponent
```

Comment:
A tego repository nie mielismy robic w oddzielnym projekcie jako dll?

Reply:
To jest do rozważenia. Co chcemy dać do oddzielnego projektu? Same interfejsy czy razem z implementacjami? Jeśli same interfejsy, to w jaki sposób będziemy korzystać z instancji interfejsu we front endzie?

Reply:
repository to na samym backendzie będą więc nie ma sensu ich wydzielać